python/clx/analytics/cybert.py

```diff
@@ -114,28 +118,21 @@ def preprocess(self, raw_data_col, stride_len=116, max_seq_len=128):
         raw_data_col = raw_data_col.str.replace("=", "= ")
         raw_data_col = raw_data_col.str.replace("\\n", " ")
 
-        byte_count = raw_data_col.str.byte_count()
-        max_rows_tensor = int((byte_count / 120).ceil().sum())
-
-        input_ids, att_mask, meta_data = raw_data_col.str.subword_tokenize(
-            self._hashpath,
-            128,
-            116,
-            max_rows_tensor=max_rows_tensor,
-            do_lower=False,
-            do_truncate=False,
+        output = self.tokenizer(
+            raw_data_col,
+            max_length=128,
+            stride=12,
```

Comment:
We changed stride here to match hugging face. 
  https://github.com/rapidsai/cudf/blob/5b8895dc50a5daf99bafede67617e8ebd39d3af6/python/cudf/cudf/core/subword_tokenizer.py#L207